source/documentors/references/quick_reference_rst.rst

```diff
@@ -19,6 +19,55 @@ Headings
 
    RST allows you to use almost any symbol to underline headings, as long as you're consistent between heading level. However, the way shown above is how headings should be defined in all Open edX documentation.
 
+Develop Sections
+****************
+
+You can nest sections in the topic as needed, to give it structure and break it up into discrete parts.
+
+Copy the Topic and Section Structure as needed
+
+.. code-block:: RST
+
+   Topic Title
+   ###########
+
+   Introduce the topic
+
+   If this is a long topic with multiple sections, use the **contents** directive below:
+
+   .. contents:: Contents
+   :local:
```

Comment:
This is indented wrong I think, the `:` needs to be two spaces in from the above `..`. I'd also add the `:depth:` parameter. 

```
.. contents::
  :depth: 1
  :local:
```